package.nix

```diff
@@ -334,6 +335,10 @@ in {
 
   hardeningDisable = lib.optional stdenv.hostPlatform.isStatic "pie";
 
+  passthru = lib.optionalAttrs (stdenv.buildPlatform.canExecute stdenv.hostPlatform) {
+    perl-bindings = nix-perl-bindings;
+  };
+
```

Comment:
I think I would rather not do that here (it doesn't work with the split packages so well), and instead just slap on the attribute for the `nix-serve` override.